src/NBS.Appointments.Service.Api.Tests/GetAvailabilityApiTests.cs

```diff
@@ -1,54 +1,97 @@
 ﻿using FluentAssertions;
 using Newtonsoft.Json;
+using System.Net;
 using System.Text;
 using Xunit;
+using NBS.Appointments.Service.Models;
 
 namespace NBS.Appointments.Service.Api.Tests
 {
     public class GetAvailabilityApiTests
-    {        
+    {
+        private readonly HttpClient _httpClient = new();
+
+        private const string BaseUrl = "http://localhost:4000/availability";
+        private const string QueryUrl = $"{BaseUrl}/query";
```

Comment:
I think we should have one test file per endpoint, but we can change that going forward